srcpkgs/elvish/template

```diff
@@ -11,12 +11,17 @@ license="BSD-2-Clause"
 homepage="https://elv.sh/"
 changelog="https://elv.sh/blog/${version}-release-notes.html"
 distfiles="https://github.com/elves/elvish/archive/v${version}.tar.gz"
-checksum=6a6006015f44def98676eaed611702b000d66838c0e76da572d517d9bde5c388
+checksum=3a4b93c3c99fe2f9847de35d64be24e2d4b9c12d429cd9831b4571993a66bb7a
 register_shell="/bin/elvish"
 
 # fix: collect2: fatal error: cannot find 'ld'
 LDFLAGS="-fuse-ld=bfd"
 
+do_check() {
```

Comment:
set `make_check=no` instead (see [Manual.md](https://github.com/void-linux/void-packages/blob/master/Manual.md) for more info)

Reply:
okay